idaes/power_generation/costing/tests/test_NGCC_costing.py

```diff
@@ -0,0 +1,352 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Case B31A - Natural Gas Combined Cycle (NGCC) plant_gross_power
+Reference: NETL Baseline Report Rev 4
```

Comment:
Provide the full reference for the baseline report (and maybe even a url if you have one).